src/patternfly/components/Scrollspy/examples/scrollspy.md

```diff
@@ -0,0 +1,362 @@
+---
+id: Scrollspy
+section: components
+cssPrefix: pf-c-scrollspy
+beta: true
+---
+
+## Examples
+### Horizontal default
+```hbs
+{{#> scrollspy}}
+  {{#> scrollspy-list}}
+    {{#> scrollspy-item}}
+      {{#> scrollspy-link}}
+        {{#> scrollspy-item-text}}
+          Inactive section
+        {{/scrollspy-item-text}}
+      {{/scrollspy-link}}
+    {{/scrollspy-item}}
+    {{#> scrollspy-item scrollspy-item--modifier="pf-m-current"}}
+      {{#> scrollspy-link}}
+        {{#> scrollspy-item-text}}
+          Active section
+        {{/scrollspy-item-text}}
+      {{/scrollspy-link}}
+    {{/scrollspy-item}}
+    {{#> scrollspy-item}}
+      {{#> scrollspy-link}}
+        {{#> scrollspy-item-text}}
+          Inactive section
+        {{/scrollspy-item-text}}
+      {{/scrollspy-link}}
+    {{/scrollspy-item}}
+  {{/scrollspy-list}}
+{{/scrollspy}}
+```
+
+### Horizontal with background
+```hbs
+{{#> scrollspy scrollspy--modifier="pf-m-background pf-m-center"}}
+  {{#> scrollspy-list}}
+    {{#> scrollspy-item}}
+      {{#> scrollspy-link}}
+        {{#> scrollspy-item-text}}
+          Inactive section
+        {{/scrollspy-item-text}}
+      {{/scrollspy-link}}
+    {{/scrollspy-item}}
+    {{#> scrollspy-item scrollspy-item--modifier="pf-m-current"}}
+      {{#> scrollspy-link}}
+        {{#> scrollspy-item-text}}
+          Active section
+        {{/scrollspy-item-text}}
+      {{/scrollspy-link}}
+    {{/scrollspy-item}}
+    {{#> scrollspy-item}}
+      {{#> scrollspy-link}}
+        {{#> scrollspy-item-text}}
+          Inactive section
+        {{/scrollspy-item-text}}
+      {{/scrollspy-link}}
+    {{/scrollspy-item}}
+  {{/scrollspy-list}}
+{{/scrollspy}}
+```
+
+### Horizontal with label
+```hbs
+{{#> scrollspy}}
+  {{#> scrollspy-main}}
```

Comment:
Currently it doesn't look like `__main` is necessary, though I'm good with adding it. If we keep it, I wonder if it could just always be present, and set to `display: flex; flex-direction: column;`. That will also keep react styles happy since it will have CSS.

Reply:
yep just added those styles actually